### PR TITLE
doc: CUDA_ERROR_FILE_NOT_FOUND

### PR DESCRIPTION
**Pull request by matthiasdiener**
A bit bleak, but that's all we have at the moment.

---

**Questions for the review**:
- [ ] Is the scope and purpose of the PR clear?
  - [ ] The PR should have a description.
  - [ ] The PR should have a guide if needed (e.g., an ordering).
- [ ] Is every top-level method and class documented? Are things that should be documented actually so?
- [ ] Is the interface understandable? (I.e. can someone figure out what stuff does?) Is it well-defined?
- [ ] Does the implementation do what the docstring claims?
- [ ] Is everything that is implemented covered by tests?
- [ ] Do you see any immediate risks or performance disadvantages with the design? Example: what do interface normals attach to?

### CODE REVIEW

**Review by MTCam — APPROVED** [review]
👍  Should it be mentioned that the probability of getting this increases with rank count?

**Comment by matthiasdiener** [comment]
> 👍 Should it be mentioned that the probability of getting this increases with rank count?

Good idea, done in [aea608c](https://github.com/illinois-ceesd/mirgecom/pull/742/commits/aea608cb4493cea7a8e2352534db2f2939e90a3f)

File: doc/faq/errors.rst [hunk 1]
```diff
@@ -68,3 +68,12 @@ to run the simulation::
...
 Try running on more nodes and/or devices.
+
+
+What can I do about ``CUDA_ERROR_FILE_NOT_FOUND: file not found`` errors?
+-------------------------------------------------------------------------
+
+The reason for these errors is unknown, they seem to be more common with higher
+rank counts. A theory is that there is a race condition between caching
+a compiled kernel on disk and the loading of that kernel from that cache.
+The only known "fix" at the moment is to restart the application.
```
Comment (inducer) [inline doc/faq/errors.rst:79]: Do you know which CUDA call issues that error code?

Comment (inducer) [reply doc/faq/errors.rst:79]: Also, it sounds like we should have a tracking issue for this problem, even if it's intermittent.

Comment (matthiasdiener) [reply doc/faq/errors.rst:79]: > Do you know which CUDA call issues that error code?

No, this is fiendishly hard to debug as it is completely spurious :-/

Comment (matthiasdiener) [reply doc/faq/errors.rst:79]: > Also, it sounds like we should have a tracking issue for this problem, even if it's intermittent.

I looked again and it turns out we have already an issue, although with not much info: https://github.com/illinois-ceesd/mirgecom/issues/679

Comment (inducer) [reply doc/faq/errors.rst:79]: Thanks for digging up the tracking issue!

**Pull request merged**